pkg/schedulingtypes/replicascheduler.go

```diff
@@ -190,6 +191,34 @@ func (s *ReplicaScheduler) Reconcile(obj pkgruntime.Object, qualifiedName ctluti
 	}
 
 	key := qualifiedName.String()
+
+	if rsp.Spec.PlacementIntersection {
+		klog.V(3).Infof("Computing placement of resource %q", qualifiedName)
+
+		fedClusters, err := s.podInformer.GetReadyClusters()
```

Comment:
Is this not the same names that you already have at line 167 above (`clusterNames, err := s.clusterNames()`)?

Reply:
you're right. I will update it.